src/lib/index.js

```diff
@@ -38,7 +38,7 @@ const getValidatedData = (data, optOuts) => {
   const missingCellRows = result[1]
 
   validatedData = _.map(validatedData, (row) => _.extend(row, {
-    cell: getFormattedPhoneNumber(row.cell) }))
+    cell: getFormattedPhoneNumber(row.cell, process.env.PHONE_NUMBER_COUNTRY || 'US') }))
```

Comment:
Great - we should update our .env template files too if we're adding another variable